ksql-engine/src/test/resources/query-validation-tests/project-filter.json

```diff
@@ -23,6 +23,19 @@
         {"topic": "S1", "key": 101, "value": "101"}
       ]
     },
+    {
+      "name": "project string with embedded code",
+      "statements": [
+        "CREATE STREAM TEST (ID bigint) WITH (kafka_topic='test_topic', value_format='DELIMITED', key='ID');",
+        "CREATE STREAM S1 as SELECT '\" + new java.util.function.Supplier<String>(){public String get() {return \"boom\";}}.get() + \"' as x  FROM test;"
+      ],
+      "inputs": [
+        {"topic": "test_topic", "key": 0, "value": "0"}
+      ],
+      "outputs": [
+        {"topic": "S1", "key": 0, "value": "\"\"\" + new java.util.function.Supplier<String>(){public String get() {return \"\"boom\"\";}}.get() + \"\"\""}
```

Comment:
A malicious user could put arbitrary java code into the expression as long as it returns a string. For example, as in this test, by constructing a supplier (with some malicious behavior in its get() call before returning some string) and getting its value.